{
  if (!getSubject().getRootCauseOfException().equals(key)) {
    fail("has root cause of exception " + key);
  }
}
